staging/src/k8s.io/component-base/logs/json/json.go

```diff
@@ -40,35 +42,39 @@ type zapLogger struct {
 	// NB: this looks very similar to zap.SugaredLogger, but
 	// deals with our desire to have multiple verbosity levels.
 	l   *zap.Logger
-	lvl zapcore.Level
+	lvl int
 }
 
 // implement logr.Logger
 var _ logr.Logger = &zapLogger{}
 
 func (l *zapLogger) Enabled() bool {
-	return l.l.Core().Enabled(l.lvl)
+	return klog.V(klog.Level(l.lvl)).Enabled()
```

Comment:
```suggestion
	return true
```
This is already checked by klog, let's skip verbosity checking in format for now.

Reply:
That means sampleInfoLogger.V(int).Info() always visible?

Reply:
yes, it should be ok, as we only intend to use JSONLogger inside klog.